nservicebus/operations/auditing.md

```diff
@@ -60,34 +60,41 @@ Audit messages can be handled in a variety of ways: Save them in a database, do
 
 partial: configuration
 
-## Audit configuration options
-
 There two settings that control auditing:
 
 ### Queue Name
 
 The queue name to forward audit messages.
 
-### OverrideTimeToBeReceived
-
-To force a [TimeToBeReceived](/nservicebus/messaging/discard-old-messages.md) on audit messages by setting `OverrideTimeToBeReceived` use the configuration syntax below.
+### Time-to-be-received
 
-Note that while the phrasing is "forwarding a message" in the implementation it is actually "cloning and sending a new message". This is important when considering TimeToBeReceived since the time taken to receive and process the original message is not part of the TimeToBeReceived of the new audit message. In effect the audit message receives the full time allotment of whatever TimeToBeReceived is used.
+The [Time-to-be-received (TTBR)](/nservicebus/messaging/discard-old-messages.md) for audit messages can be overriden.
 
-> [!WARNING]
-> MSMQ forces the same TimeToBeReceived on all messages in a transaction. Therefore, OverrideTimeToBeReceived is not supported when using the [MSMQ Transport](/transports/msmq/). If OverrideTimeToBeReceived is detected when using MSMQ an exception will be thrown with the following text:
->
-> ```
-> Setting a custom OverrideTimeToBeReceived for audits is not supported on transactional MSMQ
-> ```
+> [!NOTE]
+> What happens to messages for which the TTBR expires is different. Some transports will auto deleted expired message in the backlog of the queue while no messages are dequeued while others will only delete expired messages once the expired message reaches the front of the queue due to messages being requested. More details at [discarding old messages behavior](/nservicebus/messaging/discard-old-messages.md#behavior)
```

Comment:
I would rephrase it to "most transports will not delete". To my knowledge, it is only RabbitMQ that can actually do something useful in the audit context to prevent queue from overflowing when SC is down.

Reply:
```suggestion
> What happens to messages for which the TTBR expires is different. Most transports will **not** auto deleted expired message in the backlog of the queue and purge when dequeued and are received by the client. More details at [discarding old messages behavior](/nservicebus/messaging/discard-old-messages.md#behavior)
```